<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KcAudioCaptureDlg</class>
 <widget class="QDialog" name="KcAudioCaptureDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>录音</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>113</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>78</y>
      <width>52</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>录制格式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>44</y>
      <width>52</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>录制频率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>52</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>录制设备</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbStereo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>44</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>立体声</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>44</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDeviceList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
   </widget>
   <widget class="QComboBox" name="cbRate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>44</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMono">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>44</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>单声道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>78</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>录音品质</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFormat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>78</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <item>
     <property name="text">
      <string>Ogg Vorbis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>无损压缩FLAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PCM编码Wav</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbQuality">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>78</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="currentText">
     <string>很低</string>
    </property>
    <item>
     <property name="text">
      <string>很低</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>一般</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>正常</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>较高</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>很高</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>121</y>
     <width>341</width>
     <height>77</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>300</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="handleWidth">
     <number>5</number>
    </property>
    <widget class="QPushButton" name="btPause">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btStart">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>录制</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btStop">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btPlay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>8</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>实时音量</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTime">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>8</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:000</string>
    </property>
   </widget>
   <widget class="QtVolumnBar" name="wgVolumn" native="true">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>12</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>音频时长</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btOpenDir">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>244</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>打开录音目录</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开目录</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btSelDir">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>244</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>设置录音目录</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>设置目录...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>208</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>保存路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSaveDir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>208</y>
     <width>272</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>path</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QtWaveBar" name="wgWave" native="true">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>238</y>
     <width>150</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>btOpenDir</zorder>
  <zorder>btSelDir</zorder>
  <zorder>label_6</zorder>
  <zorder>lbSaveDir</zorder>
  <zorder>wgWave</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtVolumnBar</class>
   <extends>QWidget</extends>
   <header location="global">gui/QtVolumnBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtWaveBar</class>
   <extends>QWidget</extends>
   <header location="global">gui/QtWaveBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
